# **Lab protocol for finding primers for *Pocillopora* hard coral.**

## Stage 1:

- <https://www.ncbi.nlm.nih.gov>
- In NCBI database: search target genes sequence in *Pocillopora*.
- Scroll down ‘All Databases’ and select ‘Nucleotide’.
- Open a few sequences of a specific gene of *Pocillopora* in the ‘FASTA’ format.
- Save a few genes nucleotide sequences in word or text document. 
- Gene selected: JM ATP synthase.

## Stage 2:

- <https://www.genome.jp/tools-bin/clustalw>
- Align sequences of genes saved by copy them into ‘Clustal’ 
- Choose the option: DNA.
- Click on: “Execute Multiple Alignment”.
- The Clustal results output download and save. 
- Results of genes alignment: 
- KF583942.1      TTTTGGCAGGATTTGGTCTGGTTTTTAAATTTGCAATGTTTATAATGGTGTTTATTACGC
- KF583939.1      TTTTGGCAGGATTTGGTCTGGTTTTTAAATTTGCAATGTTTATAATGGTGTTTATTACGC
- KF583937.1      TTTTGGCAGGATTTGGTCTGGTTTTTAAATTTGCAATGTTTATAATGGTGTTTATTACGC
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      TATTAGAGGTGGCAGTTGCAATAATACAGGCCTATGTGTTTTGTCTGTTGGTACAAATTT
- KF583939.1      TATTAGAGGTGGCAGTTGCAATAATACAGGCCTATGTGTTTTGTCTGTTGGTACAAATTT
- KF583937.1      TATTAGAGGTGGCAGTTGCAATAATACAGGCCTATGTGTTTTGTCTGTTGGTACAAATTT
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      ATTTAACTGATACAATTTTTTTACATTAAAAATGACAACCGTAGAAATTTTATTATGCAC
- KF583939.1      ATTTAACTGATACAATTTTTTTACATTAAAAATGACAACCGTAGAAATTTTATTATGCAC
- KF583937.1      ATTTAACTGATACAATTTTTTTACATTAAAAATGACAACCGTAGAAATTTTATTATGCAC
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      ATTTTTAGGTATATCGTTAACGGTGAATTATGGATTTGGTATTGGGTGATATGGTAGTTC
- KF583939.1      ATTTTTAGGTATATCGTTAACGGTGAATTATGGATTTGGTATTGGGTGATATGGTAGTTC
- KF583937.1      ATTTTTAGGTATATCGTTAACGGTGAATTATGGATTTGGTATTGGGTGATATGGTAGTTC
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      TTCTCAAGCCTGGTTAAGGCACCAAATGGGCATAGACGGTGCAGATATCGTCCATCCTTG
- KF583939.1      TTCTCAAGCCTGGTTAAGGCACCAAATGGGCATAGACGGTGCAGATATCGTCCATCCTTG
- KF583937.1      TTCTCAAGCCTGGTTAAGGCACCAAATGGGCATAGACGGTGCAGATATCGTCCATCCTTG
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      AGCTTATTTTAATTATGTGCGCGAGCATTATATTGCTCGTAGTGTTGGGGTACAAGTGAA
- KF583939.1      AGCTTATTTTAATTATGTGCGCGAGCATTATATTGCTCGTAGTGTTGGGGTACAAGTGAA
- KF583937.1      AGCTTATTTTAATTATGTGCGCGAACATTATATTGCTCGTAGTGTTGGGGTACAAGTGAA
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\* \*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      TCTAGATTCAACTGAAGGACCAAATGATGTTGGGGTACAAGTGAATCCAGAGTCTGAACA
- KF583939.1      TCTAGATTCAACTGAAGGACCAAATGATGTTGGGGTACAAGTGAATCCAGAGTCTGAACA
- KF583937.1      TCTAGATTCAACTGAAGGACCAAATGATGTTGGGGTACAAGTGAATCCAGAGTCTGAACA
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      GTCTGTGGATGTTGTTTTTATAGAATTGGAAGGGCGAAGGGTCCCCGTAAGGGGGGTGAA
- KF583939.1      GTCTGTGGATGTTGTTTTTATAGAATTGGAAGGGCGAAGGGTCCCCGTAAGGGGGGTGAA
- KF583937.1      GTCTGTGGATGTTGTTTTTATAGAATTGGAAGGGCGAAGGGTCCCCGTAAGGGGGGTGAA
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      TCTACATGTTGAGCGGCGTGGTTATTTGGATAATCGGTCTAATCAAGAAGGTGTTGAGAA
- KF583939.1      TCTACATGTTGAGCGGCGTGGTTATTTGGATAATCGGTCTAATCAAGAAGGTGTTGAGAA
- KF583937.1      TCTACATGTTGAGCGGCGTGGTTATTTGGATAATCGGTCTAATCAAGAAGGTGTTGAGAA
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      TAGTTTAGGAGATCAATTTGAAGATGTGGCTTTTTTAGGGGGGAGTACAAGTACAATTAC
- KF583939.1      TAGTTTAGGAGATCAATTTGAAGATGTGGCTTTTTTAGGGGGGAGTACAAGTACAATTAC
- KF583937.1      TAGTTTAGGAGATCAATTTGAAGATGTGGCTTTTTTAGGGGGGAGTACAAGTACAATTAC
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      ACCAGAAAGTGTAAATAGGGGCGCTCCAAGTGGTAATATAAGAAATGTATTAAGTGATTC
- KF583939.1      ACCAGAAAGTGTAAATAGGGGCGCTCCAAGTGGTAATATAAGAAATGTATTAAGTGATTC
- KF583937.1      ACCAGAAAGTGTAAATAGGGGCGCTCCAAGTGGTAATATAAGAAATGTATTAAGTGATTC
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      ATTAATAGAGAGGAGAGGCCAAGACCAAACTTTAAGTGAAATAAAGGTAGAAACACCTCC
- KF583939.1      ATTAATAGAGAGGAGAGGCCAAGACCAAACTTTAAGTGAAATAAAGGTAGAAACACCTCC
- KF583937.1      ATTAATAGAGAGGAGAGGCCAAGACCAAACTTTAAGTGAAATAAAGGTAGAAACACCTCC
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

- KF583942.1      AGGTGCTTCAAGGGGTGGCCGATGAGCCAGTATAAATCAAGGTG
- KF583939.1      AGGTGCTTCAAGGGGTGGCCGATGAGCCAGTATAAATCAAGGTG
- KF583937.1      AGGTGCTTCAAGGGGTGGCCGATGAGCCAGTATAAATCAAGGTG
- `                `\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*\*

## Stage 3:

- <https://primer3.ut.ee/>
- Search in the alignment results for similarities of genes sequences on sides mostly (marked with \*) and copy the sequence.
- Paste the source sequence into the box in “Primer3web” (ver4.1.0).
- Adjust the product size range according to the size of the specific sequence (e.g., 701-850).
- Set the melting temperature (tm): 58-62<sup>0</sup>C.
- Set the GC content: 40-60%.
- Click on “Pick Primers” to generate potential primers.
- Review the results primers and verify met all criteria.
- Order specific primers from a supplier.   
- Results: 

## **Pocillopora primers output:** 	

Using 1-based sequence positions

SEQUENCE SIZE: 764

INCLUDED REGION SIZE: 764

|**OLIGO**|**start**|**len**|**tm**|**gc%**|**any\_th**|**3'\_th**|**hairpin**|**seq**|
| :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: | :-: |
|LEFT PRIMER|4|20|59\.52|50|0|0|0|TGGCAGGATTTGGTCTGGTT|
|RIGHT PRIMER|735|19|60|63\.16|0|0|0|CCCCTTGAAGCACCTGGAG|

